bindings/node/native/src/lib.rs

```diff
@@ -55,98 +55,43 @@ fn get_bot_steps(mut cx: FunctionContext) -> JsResult<JsObject> {
     Ok(js_object)
 }
 
-fn format_warnings<'a, C: Context<'a>>(
-    cx: &mut C,
-    array: &mut Handle<JsArray>,
-    warnings: Vec<Warnings>,
-) {
-    for (index, warning) in warnings.iter().enumerate() {
-        let object = JsObject::new(cx);
-        let flow = cx.string(warning.position.flow.clone());
-        let step = cx.string(warning.position.step.clone());
-        let line = cx.number(warning.position.interval.line as f64);
-        let column = cx.number(warning.position.interval.column as f64);
-        let message = cx.string(&warning.message);
-
-        object.set(cx, "flow", flow).unwrap();
-        object.set(cx, "step", step).unwrap();
-        object.set(cx, "line", line).unwrap();
-        object.set(cx, "column", column).unwrap();
-        object.set(cx, "message", message).unwrap();
-
-        array.set(cx, index as u32, object).unwrap();
-    }
-}
-
-fn format_errors<'a, C: Context<'a>>(
-    cx: &mut C,
-    array: &mut Handle<JsArray>,
-    errors: Vec<ErrorInfo>,
-) {
-    for (index, err) in errors.iter().enumerate() {
-        let object = JsObject::new(cx);
-        let flow = cx.string(err.position.flow.clone());
-        let step = cx.string(err.position.step.clone());
-        let line = cx.number(err.position.interval.line as f64);
-        let column = cx.number(err.position.interval.column as f64);
-        let message = cx.string(&err.message);
-
-        object.set(cx, "flow", flow).unwrap();
-        object.set(cx, "step", step).unwrap();
-        object.set(cx, "line", line).unwrap();
-        object.set(cx, "column", column).unwrap();
-        object.set(cx, "message", message).unwrap();
-
-        array.set(cx, index as u32, object).unwrap();
-    }
-}
-
-fn validate_bot(mut cx: FunctionContext) -> JsResult<JsObject> {
+fn validate_bot(mut cx: FunctionContext) -> JsResult<JsValue> {
     let jsbot = cx.argument::<JsValue>(0)?;
     let jsonbot: Value = neon_serde::from_value(&mut cx, jsbot)?;
 
-    let object = JsObject::new(&mut cx);
-
     match csml_engine::validate_bot(serde_json::from_value(jsonbot).unwrap()) {
         CsmlResult {
             flows: _,
             warnings,
             errors: None,
         } => {
-            let valid = cx.boolean(true);
-            object.set(&mut cx, "valid", valid).unwrap();
+            let mut value = serde_json::json!({
+                "statusCode": 200,
+                "valid": true,
+            });
```

Comment:
Can we leave this change out of this PR? Let's not change the structure of the nodejs bindings and create a separate PR please (this requires coordination with other systems).